operations/power_management/System_Power_On_Procedures.md

```diff
@@ -22,10 +22,16 @@ cabinets are powered on, wait at least 10 minutes for systems to initialize.
 
 After all the system cabinets are powered on, be sure that all management network and Slingshot network switches are powered on, and that there are no error LEDS or hardware failures.
 
-## Power on the external Lustre file system
+## Power On the External File Systems
 
 To power on an external Lustre file system (ClusterStor), refer to [Power On the External Lustre File System](Power_On_the_External_Lustre_File_System.md).
 
+To power on the external SpectrumScale (GPFS) file system, refer to site procedures.
+
+**Note:** If the external file systems are not mounted on worker nodes, then the power on of them can continue in parallel with
```

Comment:
1. Update "then the power on of them" -> "then continue to power them on in parallel..."
2. Insert a period (.) after "...compute cabinets."
3. Update "but must be completed" to "This must be completed before beginning to power on and boot..."

Reply:
fixed